nearlib/test/integration.test.js

```diff
@@ -0,0 +1,19 @@
+const account = require('../account');
+
+const aliceAccount = {
+    account_id: 'alice.near',
+    amount: 100,
+    stake: 0,
+    nonce: 0,
+    code_hash: '2Kkfab2QQ3GK7DSiuzcRF2H4vNSPKFfE9gkCDkPGumtV'
+};
+
+test('returns 1', () => {
+    // Sanity check
+    expect(account.returnOne()).toBe(1);
+});
+
+test('view account', () => {
```

Comment:
this is synchronous test, it won't work properly. Test function either should receive and call `done` callback, or use `await`:
https://jestjs.io/docs/en/asynchronous.html#async-await

Reply:
Not sure what you mean by "it won't work" - race condition? it will be flaky? It definitely works 100% of the time for me on my local if the server is running.